src/components/simulation/FooterButtons.vue

```diff
@@ -11,6 +11,9 @@ const router = useRouter()
 const simulationStore = useSimulationStore()
 
 function clickButtonCancel() {
```

Comment:
The function seems the same as in `displayErrorMessage`. Is there any reason to not make it as one function?